### PR TITLE
Feature/download img field

### PR DESCRIPTION
**Pull request by WojtekReu**

### CODE REVIEW

File: events/models.py [hunk 1]
```diff
@@ -76,15 +78,25 @@ class Meta:
     ])
 
     title = models.TextField(u'tytuł wydarzenia')
-    short_title = models.TextField(u'tytuł promocyjny wydarzenia')
+    short_title = models.TextField(u'tytuł promocyjny wydarzenia', default='', blank=True)
```
Comment (ppeczek) [inline events/models.py]: I think we shouldn't let users do that...

**Comment by ppeczek** [comment]
I can't find test_download_image method :disappointed:

File: events/models.py (hunk 1 above)
Comment (WojtekReu) [inline events/models.py]: Is it used anywhere? In any event I see "x" in this field.

Comment (ppeczek) [inline events/models.py]: It wasn't till now. Titles are crucial.

**Comment by WojtekReu** [comment]
Because there no test method for download_image. This method is not finished completely. Still missing resizing image, checking is it image, deleting old unused images.

File: events/models.py [hunk 2]
```diff
@@ -78,13 +80,23 @@ class Meta:
     title = models.TextField(u'tytuł wydarzenia')
     short_title = models.TextField(u'tytuł promocyjny wydarzenia')
 
-    title_fb_yes = models.TextField(u'tytuł na TAK obiektu FB', default='')
-    title_fb_no = models.TextField(u'tytuł na NIE obiektu FB', default='')
+    title_fb_yes = models.TextField(u'tytuł na TAK obiektu FB', default='', blank=True)
```
Comment (ppeczek) [inline events/models.py]: noooooooooooooo :D

**Comment by WojtekReu** [comment]
It's not working.

**Comment by ppeczek** [comment]
Conflicting migrations detected (0009_auto_20160504_0205, 0009_auto_20160426_2335 in events).

**Pull request closed without merging**